arbeitszeit_flask/templates/macros/draft_form.html

```diff
@@ -61,7 +61,7 @@
 <div class="box has-background-info-light">
   <div class="field">
     <div class="has-text-right has-text-info-dark">
-      {{ info_tooltip(id="id_fixed_means", text=gettext("This is sometimes also called depreciation, e.g. if you plan to use a machine that cost 10000 hours to produce, for 10 years, and you have ten one-year-plans, then for this fixed mean of production, the value would be 1000."))}}
+      {{ info_tooltip(id="id_fixed_means", text=gettext("This field is mainly to account for depreciation of machines. E.g. if you plan to use a machine that cost 10000 hours to produce, for 10 years, and you have ten one-year-plans, then for this fixed mean of production, the value would be 1000."))}}
```

Comment:
I am wondering if depreciation is the correct word or if we need a less monetary one (write-off?). The German word "Wertminderung" is incorrect IMO. I think I prefer to refer to the material side of this process (wear and tear/Abnutzung).

-> "This field is mainly to account for wear and tear of machines."
-> "Dieses Feld wird hauptsächlich dafür genutzt, der Abnutzung von Maschinen Rechnung zu tragen."

Moreover there is a small typo:
"a machine that cost 10000 hours to produce"
-> "a machine that costS 10000 hours to produce"